products/views.py

```diff
@@ -1,2 +1,43 @@
+from django.http      import JsonResponse
+from django.views     import View
+from django.db.models import Avg, F
 
+from products.models  import ProductGroup
 
+class ProductGroupView(View):
+    def get(self, request):
+        try :
+            sub_category_id  = request.GET.get("SubCategoryId")
+            product_group_id = request.GET.get("ProductGroupId")
```

Comment:
상품 목록이 아니라 상품 데이터 하나만 반환하는 경우에는 path parameter를 사용해서 구현해주세요!

```suggestion
    def get(self, request, id):
        try :
```